### ISSUE TITLE
[Bug]: Tree with actions - iOS/Android -  Figure out navigation pattern for actions button 

### ISSUE THREAD

**Issue by kolaps33** [microsoft/fluentui#28876]
### Library

React Components / v9 (@fluentui/react-components)

### System Info

```shell
System: Androind/iOS

```

### Are you reporting Accessibility issue?

None

### Reproduction

https://react.fluentui.dev/?path=/docs/preview-components-tree--default#actions

### Bug Description

Example:
https://react.fluentui.dev/?path=/docs/preview-components-tree--default#actions

## Actual Behavior
Tree with actions - tested with Talkback 
3.1 invoking context menu is possible but hard - Talkback has gesture(Double-tap and hold) with this one I got context menu only twice from about 10 tries  
VoiceOver/iOS - invoking the context menu via double-tap and hold on a tree item which has a context menu does not do anything.
3.1 when Talkback focus moves on the treeitem, then buttons are not visible, then are not navigable

## Expected Behavior
we should figure out patter what would be the best experience for android/iphone users 

### Requested priority

Blocking

### Are you willing to submit a PR to fix?

yes

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.
